index.html

```diff
@@ -0,0 +1,193 @@
+                    <div><img class="lang-img" src="images/logo2.svg" alt="csslogo"></div>
+                    <div class=""><h2 class='h2class'>Uganda</h2></div>
+                    <div class=""><p class="pclass">African dance refers to </p></div>
+                </div>
+                <div class="main-program-box col-lg-2 col-md-2 col-sm-12 d-flex mb-3 align-items-center ">
+                    <div><img class="lang-img" src="images/logo3.svg" alt="htmllogo"></div>
+                    <div class=""><h2 class='h2class'>Kenya</h2></div>
+                    <div class=""><p class="pclass">African dance refers  </p></div>
+                </div>
+                <div class="main-program-box col-lg-2 col-md-2 col-sm-12 d-flex mb-3 align-items-center box-border">
+                    <div><img class="lang-img" src="images/logo4.svg" alt="htmllogo"></div>
+                    <div class=""><h2 class='h2class'>South Africa</h2></div>
+                    <div class=""><p class="pclass">African dance refers </p></div>
+                </div>
+                <div class="main-program-box col-lg-2 col-md-2 col-sm-12 d-flex mb-3 align-items-center ">
+                    <div><img class="lang-img" src="images/logo5.svg" alt="htmllogo"></div>
+                    <div class=""><h2 class='h2class'>Zambia</h2></div>
+                    <div class=""><p class="pclass">African dance refers . </p></div>
+                </div>
+                <div class=" col-lg-2 col-md-2 col-sm-12 d-flex mb-3 align-items-center ">
+                    <button class="main-program-btn">Register</button>
+                </div>
+                <div class=" col-lg-2 col-md-2 col-sm-12 d-flex mb-3 align-items-center seeprogramclass">
+                <h2>SEE THE WHOLE PROGRAMME</h2>
+                </div>
+            </div>
+        </div>
+
+    </section>
+
+    <!-- Feature section start from here -->
+
+    <section id="feature-section" class="container-fluid feauture-program">
+  
+    </section>
+
+     <!-- Partner section start from here -->
+
+    <section id="partner-section" class="container-fluid partner-section">
+        <div class="container partner-program-container ">
+            <div class="row">
+                <div class="col-12 d-flex justify-content-center partner-main-text">Partner</div>
+            </div>
+            <div class="row d-flex justify-content-center">
+                <div class="col-1 partner-main-text-line">line</div>
+            </div>
+        </div>
+
+        <div class="container partner-img-container">
+            <img class="img1" src="images/harvardlogo.png" alt="mozilla">
+            <img class="img2" src="images/europeunilogo.png" alt="airbnb">
+            <img class="img3" src="images/Applelogo.png" alt="daumkakao">
+            <img class="img4" src="images/mercedes1.png" alt="daumkakao">
+
+        </div>
```

Comment:
- [ ] Please make sure your partner section doesn't overlap the over sections.

**Your design**
![image](https://user-images.githubusercontent.com/74411135/235280849-97e84578-c609-46b8-beb2-84d5f5bd350e.png)